src/app/asset-page/asset-page.component.ts

```diff
@@ -922,20 +923,20 @@ export class AssetPage implements OnInit, OnDestroy {
       item_id: assetId,
       additional_fields: {
         has_access: hasAccess,
-        reason_for_authorization: [reasonForAuth]
+        reason_for_authorization: [reasonForAuth],
+        ...doi && {doi: doi}
```

Comment:
This line only add the doi if it's not null?

Reply:
That is correct!